modules/ROOT/pages/installing-mobile-services.adoc

```diff
@@ -100,6 +100,15 @@ the defaults are the safest option.
 The *Wildcard DNS Host* option alters the wildcard DNS host you wish
 to use.
 
+. Check the Cluster IP:
++
+In order to allow communication between your mobile device and the cluster, the
+cluster IP must be in the same as your local machines IP address.
++
+i.e. If your workstation has the IP address: "192.168.0.2" and your mobile device 
+has the IP `192.168.0.3` then you should set the cluster IP to: `192.168.0.2` so 
```

Comment:
the IP `192.168.0.3`, ~~then you should~~ set the cluster IP to: `192.168.0.2` so